src/main/kotlin/se/uulm/snowballr/backend/auth/DummyUser.kt

```diff
@@ -20,4 +21,13 @@ object DummyUser {
     var passwordHash: String = PasswordUtils.hashPassword(password)
     var role: UserOuterClass.UserRole = UserOuterClass.UserRole.USER_ROLE_ADMIN
     var status: UserOuterClass.UserStatus = UserOuterClass.UserStatus.USER_STATUS_ACTIVE
+
```

Comment:
Either remove the empty line or put a comment there
```suggestion
    // user settings
```